src/common/user-store/user-store.ts

```diff
@@ -171,6 +177,11 @@ export class UserStore extends BaseStore<UserStoreModel> /* implements UserStore
         this[key] = newVal;
       }
     }
+
+    // TODO: Switch to action-based saving instead saving stores by reaction
```

Comment:
Why is that better?

Reply:
Because saving something should be controlled. Now it's completely uncontrolled. 

For example:

1. Get default values from file system
2. Set default values to observables
3. It shouldn't write the values to file system since they are already same

Of course I could setup the reaction after getting & setting the default values, but that's just one more timing related thing to worry which is not welcome.

Reply:
If we decide to store the values in same file, then sure writing and reading should happen from single place. However, that doesn't mean we should use any reactions for triggering the writing. Actions are more explicit, controlled and less prone for any hard-to-debug errors.

Reply:
Resolved for later possible conversation elsewhere.